packages/shared/components/popups/ManageVotingPowerPopup.svelte

```diff
@@ -97,15 +100,10 @@
         <TextHint info text={localize('popups.manageVotingPower.hint')} />
     </div>
     <div class="flex flex-row flex-nowrap w-full space-x-4">
-        <Button outline classes="w-full" onClick={onCancelClick}>
+        <Button outline disabled={isTransferring} classes="w-full" onClick={onCancelClick}>
```

Comment:
Why would we want to disable the cancel button that closes the popup if the user can click the "X" anyway?

Reply:
Because cancel to the user means that it will cancel an action, if you have already kicked off the action then the cancel button is not cancelling anything.

So its confusing to the user if it doesn't do anything. Why would we have a cancel button that only closes a popup when there is a close button.